### ISSUE TITLE
[stable/traefik] Latest chart does not provide a valid SSL from Lets Encrypt

### ISSUE THREAD

**Issue by neumachen** [helm/charts#8499]
**Is this a request for help?**: Yes

---

**Is this a BUG REPORT or FEATURE REQUEST?** (choose one): Neither

**Version of Helm and Kubernetes**:

**Which chart**: traefik

**What happened**: I can't seem to configure traefik properly to generate a valid SSL certificate from Let's Encrypt.

**What you expected to happen**: A valid SSL certificate is generated.

**How to reproduce it** (as minimally and precisely as possible):

**Anything else we need to know**:

Chart values.yml

```
traefik:
  ssl:
    enabled: false
    enforced: false

  acme:
    enabled: true
    email: foo@email.com
    staging: true
    logging: true
    challengeType: tls-alpn-01

    domains:
      enabled: false

    persistence:
      enabled: true
      ## acme data Persistent Volume Storage Class
      ## If defined, storageClassName: <storageClass>
      ## If set to "-", storageClassName: "", which disables dynamic provisioning
      ## If undefined (the default) or set to null, no storageClassName spec is
      ##   set, choosing the default provisioner.  (gp2 on AWS, standard on
      ##   GKE, AWS & OpenStack)
      ##
      # storageClass: "-"
      accessMode: ReadWriteOnce
      size: 1Gi

  dashboard:
    enabled: true
    domain: traefik-traefik-dashboard.kube-system.svc.cluster.local

  gzip:
    enabled: false

  nodeSelector:
    cloud.google.com/gke-nodepool: "default-pool"

  accessLogs:
    enabled: true
    format: json
```

**Comment by hex6502** [comment]
For the *challengeType* **tls-alpn-01** the chart needs to be generating
```
[acme]
# ...
entryPoint = "https"
[acme.tlsChallenge]
```
with the **configmap.yaml** template, but isn't - see https://docs.traefik.io/configuration/acme/#tlschallenge

**Comment by hex6502** [comment]
Just noticed the missing [acme.tlsChallenge] was fixed #8128

**Comment by PatrickHuetter** [comment]
when will be the next release that fixes that?
